### PR TITLE
Zoom selected enhancement

### PR DESCRIPTION
**Pull request by ZacZhangzhuo**
### What type of change is this?

- [x] New feature: **Zoom Selected**. Currently, the key is `F`. 

### Checklist

- [x] I added a line to the `CHANGELOG.md` file in the `Unreleased` section under the most fitting heading (e.g. `Added`, `Changed`, `Removed`).
- [x] I ran all tests on my computer and it's all green (i.e. `invoke test`).
- [x] I ran lint on my computer and there are no errors (i.e. `invoke lint`).

### CODE REVIEW

**Comment by ZacZhangzhuo** [comment]
This fixes #111 #158 #101

File: src/compas_view2/views/view.py [hunk 1]
```diff
@@ -284,6 +284,10 @@ def keyPressEvent(self, event):
         if key == QtCore.Qt.Key_Control:
             self.app.selector.mode = "deselect"
             self.keys["control"] = True
+        if key == QtCore.Qt.Key_F and self.app.selector.selected:
+            self.keys["F"] = True
```
Comment (Licini) [inline src/compas_view2/views/view.py:288]: why is F here Upper case but in init it is small case, double check?

**Review by Licini — COMMENTED** [review]
Can we also make it zoom to extents of all objects if nothing is selected? thx!

File: src/compas_view2/views/view.py (hunk 1 above)
Comment (ZacZhangzhuo) [reply src/compas_view2/views/view.py]: Done, please review

**Pull request merged**